workflows/TW-phoSim-r3/helpers/updateGit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+taskName=TW-phoSim-r3
+pipelineRoot=/nfs/farm/g/desc/u1/Pipeline-tasks/${taskName}
+
+GLOBIGNORE="*"
+src="${pipelineRoot}/config/"
+dst=${pipelineRoot}/Twinkles/workflows/${taskName}/
+
+echo src "${src}"
+echo dst ${dst}
+
+cmd="rsync -avuzb  --exclude ""*~"" ${src} ${dst}"
```

Comment:
Re: reference copy      Thanks, excellent suggestion.  Done.

Re: small data files      Agreed.  But in the interests of getting the show on the road, I'd like to postpone this until the next go 'round.